packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -231,7 +229,7 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
 	// this test relies on an internal function that has been renamed (snapshot -> summarize)
 	it("loads from snapshot", async function () {
 		// GitHub Issue: #9534
-		if (provider.driver.type === "odsp") {
+		if (provider.driver.type === "odsp" || provider.driver.type === "local") {
```

Comment:
i would add an array to this file: `const driversThatSupportBlobs=["odsp", "local"]`

then modify the checks to be `if(!driversThatSupportBlobs.includes(provider.driver.type)`

Reply:
see instead: https://github.com/microsoft/FluidFramework/pull/14291/files#r1137583078